0800-appendix-198215-dale.Rmd

```diff
@@ -41,6 +41,8 @@ a maximum of `r fpr::fpr_my_wshd(col = 'elev_max')`m to
 \@ref(tab:tab-wshd-198215)). At the time of reporting, there was no fisheries information available
 for the area upstream of crossing `r as.character(my_site)`[@moe2020KnownBC; @norris2022smnorrisbcfishobs].
 
+The site was revisited in 2023 and electrofishing was conducted upstream and downstream of the crossing. During the time of assessment, one culvert had little flowing water while the other had moderate, a significant outlet drop, and deep outlet pool.
```

Comment:
a bit more background here makes sense to me since the conclusion before says: "Electrofishing upstream and downstream of the culvert could be conducted to to provide insight into fish community composition and density to help further inform prioritization."

when updating we should move and alter this to be in the introduction as part of telling the story of why the site was revisited.

Then we need to update the conclusion to discuss what the results mean.  I haven't looked at the data but from what i remember - what we see in the results is that there are tons of salmon downstream of the culverts and exactly zero captured above confirming that the culverts are indeed a true blue barrier to fish passage.